### DISCUSSION TITLE
Shared album activity for all photos when there is a partner

### DISCUSSION THREAD

**Discussion by fivestones** [immich-app/immich#4810] [Feature Request]
### The feature

Now that Shared album activity has been implemented, it would be great to be able to make use of this functionality on all photos shared with (or from) a partner. As far as I can tell, currently it only works if I share an album with someone, and not for partner sharing. However, it would be great to have a record of comments for photos between my wife and I about photos as time goes on.

For this to work, I think partner sharing would have to be able to show all the photos of myself and my partner in the main timeline view (something that also has been separately requested) and then an option should be in place to show shared album activity on photos directly in the timeline (instead of only seeing this when viewing photos as part of a shared album). Otherwise, I'm not sure how I would see comments my wife makes on my photos (or make comments back). Or maybe there's some other way to integrate shared album activity with partner sharing.

@martabal @jrasm91 

Thanks for considering!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by zerion87** [comment]
Yes, i want this feature also. But not only partner. Share Picture between Users ANDadd them to the gallery, like pictures of the children for example.

**Comment by tomgade09** [comment]
I would also like to see this feature.
